### PR TITLE
Added ability to set Users as contact/owners of the policy when created/updated

### PR DESCRIPTION
**Pull request by rvelaVenafi**
Added ability to set the Users of the PolicySpecification file as follows:
TPP: Contacts for the Policy folder that is created.
VaaS: Owners of the Application where the Certificate Issuing Template is being added.

### CODE REVIEW

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java [hunk 1]
```diff
@@ -179,27 +173,106 @@ private static void addCitToApp(CertificateIssuingTemplate cit, Application appl
...
+		List<Application.OwnerIdsAndType> ownersList = new ArrayList<>();
+
+		if (usersList == null) {
+			// When no user is provided on the list, adds the current one as owner
+			UserDetails userDetails = cloud.authorize(apiKey);
+			String userId = userDetails.user().id();
+			Application.OwnerIdsAndType currentOwner = new Application.OwnerIdsAndType();
+			currentOwner.ownerId(userId);
+			currentOwner.ownerType(CloudConstants.OWNER_TYPE_USER);
+			ownersList.add(currentOwner);
```
Comment (marcos-albornoz) [inline src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java:209]: Please extract it in a new method

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java [hunk 2]
```diff
@@ -179,27 +173,106 @@ private static void addCitToApp(CertificateIssuingTemplate cit, Application appl
...
+								Application.OwnerIdsAndType owner = new Application.OwnerIdsAndType();
+								owner.ownerId(t.id());
+								owner.ownerType(CloudConstants.OWNER_TYPE_TEAM);
+								ownersList.add(owner);
+								break;
+							}
+						}
+					}
+
+				}
```
Comment (marcos-albornoz) [inline src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java:239]: Please extract it in a new method

**Review by marcos-albornoz — CHANGES_REQUESTED** [review]

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java [hunk 3]
```diff
@@ -179,27 +173,106 @@ private static void addCitToApp(CertificateIssuingTemplate cit, Application appl
...
+				}else{
+					if (tResponse == null) {
+						tResponse = cloud.retrieveTeams(apiKey);
+					}
+					if (tResponse != null) {
+						for (Team t : tResponse.teams()) {
+							if (t.name().equals(username)) {
+								Application.OwnerIdsAndType owner = new Application.OwnerIdsAndType();
+								owner.ownerId(t.id());
+								owner.ownerType(CloudConstants.OWNER_TYPE_TEAM);
```
[the remark this answers is not in the document]
Comment (rvelaVenafi) [reply src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java:232]: Done

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java (hunk 1 above)
Comment (rvelaVenafi) [reply src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java:209]: Done

File: src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java (hunk 2 above)
Comment (rvelaVenafi) [reply src/main/java/com/venafi/vcert/sdk/connectors/cloud/CloudConnectorUtils.java:239]: The method highlighted here is the actual one that resolves the friendly names to the owners objects from VaaS. I dont think is necessary to move this piece of logic to another method.

File: src/test/java/com/venafi/vcert/sdk/connectors/tpp/TppTokenConnectorPolicyAT.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Test
+    @DisplayName("TPP - Retrieve an Identity by username")
+    public void browseIdentities() throws VCertException {
+        try{
+            IdentityEntry identity = connectorResource.connector().getTPPIdentity(username);
+            Assertions.assertEquals(username, identity.name());
+            prefixedUniversal = identity.prefixedUniversal();
+        } catch (Exception e){
+            System.out.println("");
```
[the remark this answers is not in the document]
Comment (rvelaVenafi) [reply src/test/java/com/venafi/vcert/sdk/connectors/tpp/TppTokenConnectorPolicyAT.java:31]: Done

File: src/main/java/com/venafi/vcert/sdk/connectors/tpp/TppConnectorUtils.java [hunk 5]
```diff
@@ -307,11 +307,48 @@ public static TPPPolicy populatePolicy(TPPPolicy tppPolicy, TppAPI tppAPI) throw
...
+		List<String> usersList = new ArrayList<>();
+
+		try{
+			contactResponse = tppAPI.getPolicyAttribute(new GetPolicyAttributeRequest(policyName,
+					TppPolicyConstants.TPP_CONTACT));
+		} catch (Exception e) {
+			throw new VCertException(e);
+		}
+		if (contactResponse != null && contactResponse.error() != null){
+			throw new VCertException(contactResponse.error());
```
[the remark this answers is not in the document]
Comment (rvelaVenafi) [reply src/main/java/com/venafi/vcert/sdk/connectors/tpp/TppConnectorUtils.java]: Done

**Review by marcos-albornoz — APPROVED** [review]

**Review by ricrodriguezg — APPROVED** [review]

**Review by luispresuelVenafi — APPROVED** [review]

**Pull request merged**